src/components/App.tsx

```diff
@@ -5476,7 +5476,14 @@ class App extends React.Component<AppProps, AppState> {
 
     const elements = this.scene.getElements();
 
-    const isTextNodesOnly = elements.every((element) => isTextElement(element));
+    const selectedElements = getSelectedElements(
+      this.scene.getElements(),
+      this.state,
+    );
+
+    const isTextNodesOnly = selectedElements.every((element) =>
+      isTextElement(element),
+    );
```

Comment:
Is there actually a downside to allowing mixed elements? For example when you want to copy the whole canvas texts, so that you don't need to select items one by one.